ibm_catalog.json

```diff
@@ -0,0 +1,543 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-vpc",
+      "label": "[Add-ons Beta] Cloud automation for VPC",
+      "product_kind": "solution",
+      "tags": [
+        "network_vpc",
```

Comment:
@maheshwarishikha This is a subcategory tag. We are missing a main category tag. I'm going to add the `network` tag or it will prevent us from publishing. FYI you can see all tags by running `ibmcloud catalog filter options --all`